src/test/java/com/swift/RegistryTest.java

```diff
@@ -51,8 +55,14 @@ public RegistryFormatSupportedByIban4j(RegistryFormat registryFormat) {
             return params;
         }
 
+
         @Test
         public void registryFormatExample() {
+
+            if(Arrays.asList(ignoredListRegistryFormatExampleMethod).contains(registryFormat.getCountryCode())) {
```

Comment:
We should add the filtering in the `public static Collection<Object[]> registryFormats()` method rather than here so that the test never gets called for the ignored country codes

Reply:
Thanks for explaining the effect of the code during runtime. Changes made in the recent commit.